### ISSUE TITLE
Custom fonts not displaying on frontend or backend

### ISSUE THREAD

**Issue by actionTurtle** [elementor/elementor#12726]
#  Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

## Description
 
I have noticed that fonts from Envato Elements do not work in the custom fonts section. Fonts from Dafont and other font resources seem to work fine. I have uploaded the .ttf, .woff and woff2 files into their corresponding places in custom fonts. I have flushed the cache of both my browser and server. I have tried several different browsers (all with flushed cache) I have regenerated the CSS files each time... No luck...

Side Note* These fonts come with custom glyphs (beyond just A-Z) I have searched all over the web for a way to display the custom glyphs (I have always accessed them before using photoshop or Illustrator) but can not find anything on how to use them with elementor. I assume this would be a feature request actually and I should make another thread for it?

## Steps to reproduce

Try to upload the .ttf, woff or woff2 files from the URL below. You have to have a envato elements account to download the fonts so I made a public drive folder for people to download to see if they have the same issue... Again I have only noticed this issue coming from fonts from envato... Thank you for anyones insights... 

[https://drive.google.com/drive/folders/1opliOzHZ9sviMGxacqYkDnvLC6od8UK2?usp=sharing](url)

## Isolating the problem
- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a default WordPress theme active.
- [X] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
					
== Server Environment ==
	Operating System: Linux
	Software: nginx/1.18.0
	MySQL version: MySQL Community Server - GPL v8.0.21
	PHP Version: 7.4.10
	PHP Max Input Vars: 1000
	PHP Max Post Size: 64M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.5.1
	WP Multisite: No
	Max Upload Size: 64 MB
	Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: en-US
	Debug Mode: Active

== Theme ==
	Name: Hello Elementor
	Version: 2.3.0
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator

== Active Plugins ==
	Akismet Anti-Spam
		Version: 4.1.6
		Author: Automattic

	Analytify - Google Analytics Dashboard
		Version: 3.1.5
		Author: Analytify

	Classic Editor
		Version: 1.6
		Author: WordPress Contributors

	Elementor
		Version: 3.0.11
		Author: Elementor.com

	Elementor Pro
		Version: 3.0.5
		Author: Elementor.com

	Rank Math SEO
		Version: 1.0.48.2
		Author: Rank Math

	Wordfence Security
		Version: 7.4.11
		Author: Wordfence


== Must-Use Plugins ==
	

== Elements Usage ==
	
	header : 1
		icon-list : 1
		image : 1
		nav-menu : 1
	popup : 1
		image : 1
	wp-page : 1
		heading : 1
		spacer : 2
		text-editor : 1


== Log ==
	
Log: showing 1 of 12020-10-02 03:25:44 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.0.4',
  'to' => '3.0.5',
)]


				
```
</details>

**Comment by udidol** [comment]
Hi @actionTurtle,
I am not sure I understood exactly what your problem is.
When you create your custom font using our interface, does it not appear in the custom fonts list in Elementor Editor?

If it does, the problem is that you try to use it but you see a different (fallback) font? Or you see squares? Or you see nothing at all?

Please be more specific regarding what your problem is, including steps to reproduce.

**Comment by johnbolandwp** [comment]
At this time for me, only one of my custom fonts is showing in the editor.  

I added 3 fonts in the same way you're supposed to, but only one is showing as an option.

**Comment by actionTurtle** [comment]
Yeah it just uses a fall back font... Rather then using the custom font I defined in the interface... 

When I upload certain fonts you can even see the font style in the elementor custom fonts section... But for some reason the fonts from envato just look like airel... or something
